src/pages/mine.tsx

```diff
@@ -1,11 +1,58 @@
-import { FC } from 'react';
+import { FC,useState, useEffect } from 'react';
 import Head from '@/components/head';
 import Layout from '@/components/Layout';
+import Card from '@/components/tasks/mine_card';
+import useFetch from '@/hooks/useFetch';
+import classNames from '@/styles/tasks.module.scss';
+import { task } from '@/components/constants/types';
 
-const Mine: FC = () => (
+const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks/self`;
+
+
+const CardList = (tasks: task[]) => tasks.map(
+    (item: task) => <Card content={item} key={item.id} />);
```

Comment:
we could shift the arrow fn body to the next line OR
put the closing paranthesis on the next line